pkg/sql/schemachanger/scop/mutation.go

```diff
@@ -541,3 +541,16 @@ type RefreshStats struct {
 	mutationOp
 	TableID descpb.ID
 }
+
+// AddColumnToIndex mutates an index to add a column to it.
+// The column should already exist on the table and so should
+// the index.
```

Comment:
Side note: One confusion I start to get as I learned about the new schema changer is "what does it mean to say a column/index exists on a table". 

When I'm learning the old schema changer, I learned that if a table descriptor is undergoing a schema change, it will have mutations queued up in them. This means, when using the table descriptor to interpret the rows, we need to consult both the table indexes/columns as well as any queuing mutations. For example, in a table descriptor, if we can find a column 'k', but we also find a drop_column_k as its mutation that is write_and_delete_only, we say "column k does not exist as far as reading is concerned".

Now, with the new schema changer, we further introduce a place to state "status" in elements. For example, when a column is added, its associated elements are marked as "absent" and "toPublic". But I think a lot of operations in the new schema changer world will just treat this column as "already exists" and proceed to operate on their elements accordingly. 

Maybe the way I am approaching to understanding this is off and is actually a lot simpler. Can you try to clear it up a bit for me? What's your mental model (or, how do you think to yourself) when something (e.g. a table/index/column) "already exists" and when it "already no longer exists".